docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -65,7 +65,9 @@ To integrate your OAuth2 provider with Grafana using our generic OAuth2 authenti
 
    b. Extend the `scopes` field of `[auth.generic_oauth]` section in Grafana configuration file with refresh token scope used by your OAuth2 provider.
 
-   c. Enable the refresh token on the provider if required.
+   c. Set `use_refresh_token` to `true` in `[auth.generic_oauth]` section in Grafana configuration file.
+
+   d. Enable the refresh token on the provider if required.
```

Comment:
```suggestion
   d. Enable refresh token on the provider if the provider requires enabling offline access.
```

something like this?

Reply:
I would leave the current one because it has been suggested by the docs team!